server/src/main/scala/org/scassandra/server/ServerStubRunner.scala

```diff
@@ -40,7 +37,14 @@ object ServerStubRunner extends LazyLogging {
     logger.info(s"Using binary port to $binaryPortNumber and admin port to $adminPortNumber")
     val ss = new ServerStubRunner(binaryListenAddress, binaryPortNumber, adminListenAddress, adminPortNumber, ScassandraConfig.startupTimeout)
     ss.start()
-    ss.awaitTermination()
+
+    // wait indefinitely or until interrupted.
+    val obj = new Object()
+    obj.synchronized { obj.wait(); }
```

Comment:
It does cause the thread to be interrupted and for the process to ultimately exit (which is desired I think).